project/src/components/app/app.jsx

```diff
@@ -1,7 +1,19 @@
 import React from 'react';
+import MainScreen from '../main-screen/main-screen';
+import PropTypes from 'prop-types';
 
-function App() {
-  return <p>Hello, world!</p>;
+function App({promoFilm, films}) {
```

Comment:
в академии принято использовать function для тех случаев, где требуется `this`. В функциональных компонентах этого делать нельзя по договоренности. На то они и функциональные компоненты, чтобы быть чистыми функциями.

Best Practice

```suggestion
const App = (p) => {
  const {promoFilm, films} = p;
```

так, чтобы видно было
  * this не используется
  * пропсы, даже после выпиливания менять знаения не будут